bluez-async/src/events.rs

```diff
@@ -47,17 +49,23 @@ pub enum AdapterEvent {
 
 /// Details of an event related to a Bluetooth device.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
 pub enum DeviceEvent {
     /// A new device has been discovered.
     Discovered,
     /// The device has connected or disconnected.
     Connected { connected: bool },
     /// A new value is available for the RSSI of the device.
     RSSI { rssi: i16 },
+    /// A new value is available for the manufacturer-specific advertisement data of the device.
+    ManufacturerData {
+        manufacturer_data: HashMap<u16, Vec<u8>>,
+    },
 }
 
 /// Details of an event related to a GATT characteristic.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
```

Comment:
what are we expecting to add in the future?

Reply:
I don't have anything particularly in mind, but it seems like a good idea in general to avoid more breaking changes.